public/list.json

```diff
@@ -0,0 +1 @@
"Vishal Chauhan","username":"vishalchauhan","avatar":"https://github.com/vish-han.png","linkCount":5},{"type":"personal","name":"Vivek Kumar Singh","username":"vivek","avatar":"https://github.com/vivekthedev.png","linkCount":2},{"type":"personal","name":"voidz","username":"voidz","avatar":"https://github.com/voidz7.png","linkCount":2},{"type":"personal","name":"Vraj Desai","username":"vrajdesai78","avatar":"https://github.com/vrajdesai78.png","linkCount":2},{"type":"personal","name":"Vasanth Srivatsa","username":"vsnthdev","avatar":"https://github.com/vsnthdev.png","linkCount":4},{"type":"personal","name":"ADEWALE ADEYEMO","username":"WaleyDevOps","avatar":"https://github.com/WaleyDevOps.png","linkCount":3},{"type":"personal","name":"Wasiq Amjad Bhamla","username":"wasiqb","avatar":"https://github.com/WasiqB.png","linkCount":3},{"type":"community","name":"Web3 Community","username":"web3community","avatar":"https://github.com/web3community.png","linkCount":8},{"type":"personal","name":"Chinwendu Enyinna","username":"wendeee","avatar":"https://github.com/wendeee.png","linkCount":4},{"type":"personal","name":"William J Priest","username":"WilliamJ","avatar":"https://github.com/WilliamJPriest.png","linkCount":4},{"type":"personal","name":"Yash Patel","username":"Yash0025","avatar":"https://github.com/yash-0025.png","linkCount":3},{"type":"personal","name":"Yashraj thakor","username":"yashrajthakor","avatar":"https://github.com/yashrajthakor.png","linkCount":1},{"type":"personal","name":"Yash Wankhade","username":"yashwankhade","avatar":"https://github.com/yashwankhade.png","linkCount":2},{"type":"personal","name":"Yashwnath Srivathsava","username":"yashwanthSRIVAT","avatar":"https://github.com/yashwanthSRIVAT.png","linkCount":3},{"type":"personal","name":"Ahmad Yahya","username":"yax-coder","avatar":"https://github.com/Yax-coder.png","linkCount":2},{"type":"personal","name":"Yogesh Summan","username":"yogesh","avatar":"https://github.com/yogeshsumman.png","linkCount":2},{"type":"personal","name":"Chad Whitt","username":"youhavechad","avatar":"https://github.com/youhavechad.png","linkCount":3},{"type":"personal","name":"Julia Undeutsch","username":"YuriDevAT","avatar":"https://github.com/yuridevat.png","linkCount":3},{"type":"personal","name":"Zabih Haqqani","username":"zabihhaqqani","avatar":"https://github.com/zabihhaqqani.png","linkCount":4},{"type":"personal","name":"Zaid Khan","username":"ZaidKhan2002","avatar":"https://github.com/ZaidKhan2002.png","linkCount":1},{"type":"personal","name":"Zehra Anwar","username":"zehra47","avatar":"https://github.com/zehra47.png","linkCount":2},{"type":"personal","name":"Zishan Mohd","username":"zishan","avatar":"https://github.com/Zishan-7.png","linkCount":2},{"type":"personal","name":"Riccardo Zichittella","username":"zodraccir","avatar":"https://github.com/zodraccir.png","linkCount":1},{"type":"personal","name":"Ahmed Zougari","username":"zougari47","avatar":"https://github.com/zougari47.png","linkCount":5}]
```

Comment:
I think this file is not needed for the nextjs branch, thanks for spotting another bug 👍